src/universal/components/Editable/Editable.js

```diff
@@ -0,0 +1,198 @@
+import React, {Component, PropTypes} from 'react';
+import withStyles from 'universal/styles/withStyles';
+import {css} from 'aphrodite-local-styles/no-important';
+import appTheme from 'universal/styles/theme/appTheme';
+import ui from 'universal/styles/ui';
+import makePlaceholderStyles from 'universal/styles/helpers/makePlaceholderStyles';
+import FontAwesome from 'react-fontawesome';
+
+class Editable extends Component {
```

Comment:
Oh! This is what you meant when you kept saying "we'll need to make this `Editable`". I didn't realize! Cool!